yt_dlp/YoutubeDL.py

```diff
@@ -3298,7 +3298,7 @@ def check_max_downloads():
             else:
                 try:
                     self.to_screen('[info] Writing video annotations to: ' + annofn)
-                    with open(annofn, 'w', encoding='utf-8') as annofile:
+                    with open(annofn, 'w', encoding='utf-8', buffering=65536) as annofile:
```

Comment:
Specifying a buffer size in text mode does nothing, according to [`open()`](<https://docs.python.org/3/library/functions.html#open>)?

Reply:
....no?

> - Interactive” text files (files for which isatty() returns True) use line buffering. Other text files use the policy described above for binary files.

Reply:
> Note that specifying a buffer size this way applies for binary buffered I/O, but TextIOWrapper (i.e., files opened with mode='r+') would have another buffering

This? Maybe I misunderstand

Reply:
It's saying that there will be ANOTHER buffer in that case. It's poorly worded. r+ is read/write so it makes sense that there would be two buffers involved, a read buffer and a write buffer.

Reply:
I should also add that the code there says annotations are deprecated anyway, plus the buffering option with any choice 8192 or up will be either beneficial or have no effect. 8192 is the default buffer size on my Windows machine. I considered making the buffer even larger, but there is a principle of diminishing returns. 8K to 64K eliminates 87.5% of all write requests already.

Reply:
> annotations are deprecated anyway

That was just a general commend about text mode. I think its fine yeah.